libs/s25main/desktops/dskSelectMap.cpp

```diff
@@ -313,7 +315,7 @@ void dskSelectMap::Msg_TableChooseItem(const unsigned /*ctrl_id*/, const unsigne
 void dskSelectMap::CreateRandomMap()
 {
     // setup filepath for the random map
-    std::string mapPath = (bfs::path(RTTRCONFIG.ExpandPath(FILE_PATHS[48])) / "Random.swd").string();
+    std::string mapPath = (bfs::path(RTTRCONFIG.ExpandPath(s25::folders::mapsPlayed)) / "Random.swd").string();
```

Comment:
should perhaps also be a separate constant

Reply:
I'd say this is a path that is local to this code only. The folder is a reusable constant but the filename is pretty much arbitrary so I'd say it can stay here.